frontend/packages/pipelines-plugin/src/utils/pipeline-utils.ts

```diff
@@ -247,6 +249,16 @@ export const getPipelineTasks = (
   return out;
 };
 
+export const getFinallyTasks = (
+  pipeline: PipelineKind,
+  pipelineRun: PipelineRunKind = {
+    apiVersion: '',
+    metadata: {},
+    kind: 'PipelineRun',
+    spec: {},
+  },
```

Comment:
Instead of mocking the pipelineRun object, call this `getFinallyTasksWithStatus` only when you have pipelineRun object.